examples/imagestamping/deployment.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     metadata:
       labels:
         app: image-stamping
+        deployed-by: "%{deployed_by}"
```

Comment:
Is it relevant to add a test to verify this is resolved correctly?

Reply:
Yes, I think it would be good to have a test. Let me think a bit more about how to test this correctly.

Reply:
hmm, It will probably need to be an e2e test. Check out how the current [e2e tests](https://github.com/bazelbuild/rules_k8s/blob/master/test-e2e.sh#L116) are set up. If its just a matter of checking the contents of an output file or running bazel run and checking its output, you can do something like what rules_docker e2e tests do ([example](https://github.com/bazelbuild/rules_docker/blob/master/testing/e2e/cc_image.sh#L26), [example](https://github.com/bazelbuild/rules_docker/blob/master/testing/e2e/docker_run_flags.sh#L28)). Let me know if you need more clarifications on any of these suggestions or if I can otherwise help.